crates/aptos-rosetta/src/account.rs

```diff
@@ -273,25 +263,27 @@ impl CoinCache {
 
         let response = if let Some(version) = version {
             rest_client
-                .get_account_resource_at_version_bcs::<CoinInfo>(
+                .get_account_resource_at_version_bcs::<CoinInfoResource>(
                     address,
                     &encoded_resource_tag,
                     version,
                 )
-                .await?
+                .await
         } else {
             rest_client
-                .get_account_resource_bcs::<CoinInfo>(address, &encoded_resource_tag)
-                .await?
+                .get_account_resource_bcs::<CoinInfoResource>(address, &encoded_resource_tag)
+                .await
         };
 
         // At this point if we've retrieved it and it's bad, we error out
-        let coin_info = response.into_inner();
+        let coin_info = response?.into_inner();
 
         // TODO: The symbol has to come from a trusted list, as the move type is the real indicator
         Ok(Some(Currency {
-            symbol: coin_info.symbol,
-            decimals: coin_info.decimals,
+            symbol: coin_info
+                .symbol()
+                .unwrap_or_else(|_| "Unknown Symbol".to_string()),
```

Comment:
The actual error from symbol implies this fails when the data is invalid (not UTF-8), not unknown. Is that right? If so could we make this like "Invalid Symbol" and include the error message.

Reply:
Yeah, we can change that now